README.md

```diff
@@ -6,7 +6,22 @@ This documentation is made up of various [arc42 structured](https://docs.arc42.o
 
 ## Systems
 
-- [Overview](./systems/overview/)
+[Overview](./systems/overview/) provides a high level view of the whole Wikidata / Wikibase system landscape.
+
+Multiple top level systems are identified.
+
+Some of these also have fairly notable sub systems identified here for convenience.
+
 - [WikibaseRepo](./systems/WikibaseRepo/)
+  - Termbox & Termbox SSR
+  - EntitySchema
+  - Property Suggester
+  - QualityConstraints
+  - WikibaseLexeme
```

Comment:
as per our discussion notes, `MediaInfo` should be added here under WikibaseRepo

Reply:
Added